.github/ISSUE_TEMPLATE/bug.yml

```
@@ -12,17 +12,23 @@ body:
           - Posting on [Discourse](https://discuss.python.org/c/users/7)
           - Reading the [Python tutorial](https://docs.python.org/3/tutorial/)
           - Emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
-  - type: checkboxes
+
+        Also make sure to search the [CPython issue tracker](https://github.com/python/cpython/issues?q=is%3Aissue+sort%3Acreated-desc) to check that the bug has not already been reported.
```

We should use the imperative mood here and direct that an action be taken
```suggestion
        Search the [CPython issue tracker](https://github.com/python/cpython/issues?q=is%3Aissue+sort%3Acreated-desc) to check that the bug has not already been reported.
```

---

("Make sure" already uses the imperative mood 😉 But I agree that we can be more assertive here!)

---

> "Make sure" already uses the imperative mood

I objected mainly to "Also"!